src/createServer.js

```diff
@@ -1,3 +1,37 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { handleErrors } = require('./handleErrors');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+
+    const toCase = normalizedURL.searchParams.get('toCase');
+    const text = normalizedURL.pathname.slice(1);
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const errors = handleErrors(text, toCase);
```

Comment:
Do you throw any errors to handle here? Use more descriptive names, this function should **validate params**, so better to call it this way